app-modules/service-management/src/Filament/Resources/ContractResource/Pages/CreateContract.php

```diff
@@ -0,0 +1,131 @@
+            ->schema([
+                TextInput::make('name')
+                    ->required(),
+                Select::make('contract_type')
+                    ->required()
+                    ->visible(ContractManagement::active())
+                    ->label('Contract Type')
+                    ->relationship('contractType', 'name', fn (Builder $query) => $query->orderBy('order', 'ASC'))
+                    ->preload()
+                    ->default(
+                        fn () => ContractType::query()
+                            ->where('is_default', true)
+                            ->first()
+                            ?->getKey()
+                    )
+                    ->native(false)
+                    ->searchable(),
+                TextInput::make('vendor_name')
+                    ->label('Vendor Name')
+                    ->required(),
+                TextInput::make('contract_value')
+                    ->label('Contract Value')
+                    ->prefix('$')
+                    ->numeric()
+                    ->step(0.01)
+                    ->minValue(0)
+                    ->rule('decimal:0,2')
+                    ->required(),
+                DatePicker::make('start_date')
+                    ->label('Start Date')
+                    ->native(false)
+                    ->displayFormat('m/d/Y')
+                    ->live(onBlur: true)
+                    ->afterStateUpdated(
+                        fn (Get $get, Set $set) => $get('start_date') > $get('end_date') ? $set('end_date', '') : ''
+                    )
+                    ->closeOnDateSelection()
+                    ->placeholder('Select start date')
+                    ->required(),
+                DatePicker::make('end_date')
+                    ->label('End Date')
+                    ->native(false)
+                    ->displayFormat('m/d/Y')
+                    ->placeholder('Select end date')
+                    ->minDate(fn (Get $get) => $get('start_date'))
+                    ->live(onBlur: true)
+                    ->closeOnDateSelection()
+                    ->required(),
+                Textarea::make('description'),
+                SpatieMediaLibraryFileUpload::make('contract_files')
+                    ->label('Contract Files')
+                    ->disk('s3')
+                    ->acceptedFileTypes([
+                        'application/pdf',
+                        'application/vnd.ms-word',
+                        'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+                    ])
+                    ->downloadable()
+                    ->maxFiles(5)
+                    ->visibility('private')
+                    ->multiple(),
+            ]);
+    }
+
+    protected function mutateFormDataBeforeCreate(array $data): array
+    {
+        $data['contract_value'] = Money::parseByDecimal($data['contract_value'], config('money.defaultCurrency'));
```

Comment:
Hi,

There is no option to set fallback currency, hence I need to pass it, If it's not pass then it will throw following error:

DecimalMoneyParser cannot parse currency symbols. Use fallbackCurrency argument.

Thank you.